README.md

```diff
@@ -27,7 +37,7 @@ When your container is built, when you open `Terminal -> New Terminal` and you'l
 You should be able to build docs:
 
 ```bash
-$ make html
+$ flux start make html
```

Comment:
noob request, instructions for how to run docker container with all the magic to allow person to run `flux start make html`.  Not just for me, but for the casual person that is just "oh i found a typo, whats the fastest way for me to fix and submit PR".

Reply:
@chu11 the README has an entire, step by step instruction for getting to this spot in the development container with VSCode - what more are you looking for here?

Reply:
i'm not a vscode developer, so i guess i was looking for "what's the fastest way I could fix a typo and type "flux start make html"?

Reply:
Don't most flux devs that do things "old school" just have flux compiled locally?

Reply:
If you want a "manual tutorial" for using a container as an environment I can add one, but separately, it's out of scope for this PR.

Reply:
I would also argue a noob making a fix would either just do a tweak in a GitHub editor, or at most pull, fix, and then push without rebuilding locally. But with the change I just made it should work either way, if they want to build or not. It will only fail the CI here if they make a change to the examples gallery without updating it. And if they are truly unable to have an environment with flux someone on the maintainer team can run it and push on their behalf (I'm happy to do that).

Reply:
that's a good point, i've been prone to doing that ... just edit the doc, push to CI and hope I didn't screw up along the way.